Fix fallibility error in example vector config (#11575)
DataDog/documentation: content/en/agent/vector_aggregation.md

    inputs:
      - datadog_agents
    source: |

>       # We use the `!` shorthand with the `string` function to error if .ddtags is not a "string"
>       # as we always expect the .ddtags field to be of type "string"

Reviewer: We don't use "we" in the docs
```suggestion
      # The `!` shorthand is used here with the `string` function, it errors if .ddtags is not a "string".
      # The .ddtags field is always expected to be a string.
```
